web/geo/generateZoneBoundingBoxes.ts

```diff
@@ -11,7 +11,7 @@ const zones = mergeZones();
 
 if (arguments_.length <= 0) {
   console.error(
-    'ERROR: Please add a zoneName parameter ("node generate-zone-bounding-boxes.js DE")'
+    'ERROR: Please add a zoneName parameter ("ts-node generateZoneBoundingBoxes.ts DE")'
```

Comment:
Will this work without ts-node?

Reply:
Actually nvm :)

Reply:
You already resolved it and yes technically it will work without ts-node for now but not when we start fixing the types.